packages/core/core-flows/src/product/steps/get-product-sales-channel-link.ts

```diff
@@ -0,0 +1,33 @@
+import { Modules } from "@medusajs/modules-sdk"
+import { ContainerRegistrationKeys } from "@medusajs/utils"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+type StepInput = {
+  ids: string[]
+}
+
+export const getProductSalesChannelLinkStepId = "get-product-sales-channel-link"
+export const getProductSalesChannelLinkStep = createStep(
```

Comment:
question: Can we use the `remoteQueryStep` instead of having a separate step for this?